gap/perm/largebase.gi

```diff
@@ -526,6 +538,7 @@ FindHomMethodsPerm.LargeBasePrimitive :=
     fi;
     RECOG.SetPseudoRandomStamp(grp,"Jellyfish");
     res := RECOG.AllJellyfish(grp);
+    RECOG.SetPseudoRandomStamp(grp,"PseudoRandom");
```

Comment:
Why not this? (Honest question, I don't know)
```suggestion
    RECOG.SetPseudoRandomStamp(grp,"LargeBasePrimitive");
```

Reply:
`LargeBasePrimitive` is more or less a wrapper for `AllJellyfish`. After the call to `AllJellyfish` the actual computations are done and now we're just managing stuff. Thus here I "reset" the stamp.

Yeah, I really do not like how we handle this whole RandomStamp business currently, see #261 for an idea how to improve this.